docs/UserGuide.md

```diff
@@ -288,6 +289,51 @@ Format: `add_task td/TASK_DESCRIPTION d/DEADLINE ev/EVENT_NAME`
 Examples:
 * `add_task td/Book Venue d/2023-12-23 ev/NUS Career Fair 2023` adds a `Book Venue` by `2023-12-23` task to the event `NUS Career Fair 2023`
 
+### Deleting a task: `delete_task`
+
+Deletes the task specified by the task description from its associated event in JobFestGo.
+
+Format: `delete_task td/TASK_DESCRIPTION ev/EVENT_NAME`
+
+* Deletes the task specified by the `TASK_DESCRIPTION` from the event specified by the `EVENT_NAME` of JobFestGo.
+* Both the `TASK_DESCRIPTION` and the `EVENT_NAME` are case-insensitive.
+ e.g. `delete_task td/Book Venue ev/NUS Career Fair 2023` and `delete_task td/book venue ev/nus Career FAIR 2023`
+ will perform the same operation.
+* Warnings will be sent if the specified event does not exist or the event does not have the specified task.
```

Comment:
should this be this instead
```suggestion
* Error will be sent if the specified event does not exist or the event does not have the specified task.
```

Reply:
I feel that error is different from a warning. A warning would mean that the system still accepts the input but reminds the user that something can go wrong, whereas an error outright prevents the input, similar to how checkstyle warnings vs errors work

Reply:
I think this is a good suggestion! Thank you!